dist/win/build.ps1

```diff
@@ -77,14 +103,42 @@ $Env:JP_WIXWIZARD_RESOURCES = "$buildDir\resources"
 	--app-image Cryptomator `
 	--dest installer `
 	--name Cryptomator `
-	--vendor "Skymatic GmbH" `
-	--copyright "(C) 2016 - 2022 Skymatic GmbH" `
+	--vendor $vendor `
+	--copyright $copyright `
 	--app-version "$semVerNo" `
 	--win-menu `
 	--win-dir-chooser `
 	--win-shortcut-prompt `
-	--win-update-url "https:\\cryptomator.org" `
+	--win-update-url $updateUrl `
 	--win-menu-group Cryptomator `
 	--resource-dir resources `
+	--about-url $aboutUrl `
 	--license-file resources/license.rtf `
 	--file-associations resources/FAvaultFile.properties
+
+#Create RTF license for bundle
+#TODO: actually we should patch also the third-party-file in the Cryptomator jar
```

Comment:
nope we shouldn't, the license shown in the app doesn't need to mention winfsp, as it is not "part of" Cryptomator

Reply:
I added this due to the following statement in the winfsp license:
https://github.com/billziss-gh/winfsp/blob/cdcd6af81d62fbdbabf9812cb4b9f83ef3540ac3/License.txt#L20-L22

Reply:
This passage applies only to non-GPLv3 projects that still wish to use WinFSP. 

In general, we only need to include copyright notices of IP that we're redistributing (as in case of the bundled installer). Otherwise you'd need to add copyright notices for every system library that you link against.

Therefore I guess it is sufficient to include the WinFSP copyright statement only in the installer that bundles WinFSP, right @billziss-gh?

Reply:
Your project is GPLv3 which is the same license that WinFsp uses. Therefore you do not need the "special exception to GPLv3" to use WinFsp.

@overheadhunter if I understand you correctly there will be 2 installers: one which bundles the WinFsp MSI and one which does not. If that is the case I agree that only the installer that includes WinFsp needs to include the WinFsp copyright notice (and a link to the WinFsp GitHub repo to satisfy section [6](https://github.com/billziss-gh/winfsp/blob/cdcd6af81d62fbdbabf9812cb4b9f83ef3540ac3/License.txt#L279) of the GPLv3).

Reply:
Since this is already the [case](https://github.com/cryptomator/cryptomator/pull/2072/files#diff-f3ce2171ed0a529e53bb10cf003ea7bf1d20d632f6d76f12036ca0c37c9264c9R40), this question should be resolved.

TODO removed in https://github.com/cryptomator/cryptomator/pull/2072/commits/467d9e56a0bdfda9229c5d5bfd7885049fa13108

Reply:
@infeo thanks!